R/license_note.R

```diff
@@ -0,0 +1,96 @@
+#' Generate LICENSE.note file.
+#'
+#' LICENSE.note generated by this function lists dependent Rust packages infomation.
```

Comment:
I am not a native English speaker, but `dependent packages` reads to me as packages that depend on the current one, while we are checking packages that the current package depends on. How about 
```suggestion
#' LICENSE.note generated by this function contains information about Rust crate dependencies.
```